components/tooltip/index.css

```diff
@@ -59,100 +133,322 @@ governing permissions and limitations under the License.
   p {
     margin: 0;
   }
+
+  /****** Variants ******/
+  &--info,
+  &--info .spectrum-Tooltip-tip {
+    background-color: var(--mod-tooltip-background-color-informative, var(--spectrum-tooltip-background-color-informative));
+  }
+
+  &--positive,
+  &--positive .spectrum-Tooltip-tip {
+    background-color: var(--mod-tooltip-background-color-positive, var(--spectrum-tooltip-background-color-positive));
+  }
+
+  &--negative,
+  &--negative .spectrum-Tooltip-tip {
+    background-color: var(--mod-tooltip-background-color-negative, var(--spectrum-tooltip-background-color-negative));
+  }
 }
 
+/****** Tip Base Styles ******/
 .spectrum-Tooltip-tip {
   position: absolute;
 
-  height: 0;
-  width: 0;
+  height: var(--mod-tooltip-tip-block-size, var(--spectrum-tooltip-tip-block-size));
+  width: var(--mod-tooltip-tip-inline-size, var(--spectrum-tooltip-tip-inline-size));
 
-  border-width: var(--spectrum-tooltip-neutral-tip-width);
-  border-style: solid;
-  border-left-color: transparent;
-  border-right-color: transparent;
-  border-bottom-color: transparent;
+  top: 100%;
+  left: 0;
+  right: 0;
+  margin: auto;
+
+  background-color: var(--mod-tooltip-background-color-default, var(--spectrum-tooltip-background-color-default));
```

Comment:
Should we add the `--highcontrast-` var here, too?

Reply:
Yes, please. IIUC, this would correct: https://d616394b12747a75b01ea075edf43b3d--spectrum-web-components.netlify.app/review/